static/app/views/codecov/tests/preOnboarding.tsx

```diff
@@ -0,0 +1,207 @@
+            )}
+            <Link to="https://docs.codecov.com/docs/test-analytics">
+              {t('Learn more')}
+            </Link>
+          </StyledP>
+          <ItalicP>
+            {t(
+              'No more staring at failing tests wondering, “Is it me, or is it just flaky?”'
+            )}
+          </ItalicP>
+        </div>
+      </IntroSection>
+      <InstructionsSection>
+        <h2>{instructionSet.header}</h2>
+        <p>{instructionSet.subtext}</p>
+        <ButtonBar>
+          <LinkButton priority="primary" href={instructionSet.mainCTA}>
+            {instructionSet.mainCTA}
+          </LinkButton>
+          <LinkButton priority="default" href="/settings/integrations/github">
+            Learn more
+          </LinkButton>
+        </ButtonBar>
+        <PrerequisitesSection>
+          <PrerequisitesTitle>
+            {t('Prerequisites to connect your GitHub organization:')}
+          </PrerequisitesTitle>
+          <Prerequisites>
+            <Prereq>
+              <PrereqMainText>{t('Enable GitHub as an Auth Provider')}</PrereqMainText>
+              <PrereqSubText>
+                {t(
+                  "Sentry Prevent analyzes your code through your Git provider. You'll need to authenticate to access data from your organizations."
+                )}
+              </PrereqSubText>
+            </Prereq>
+            <Prereq>
+              <LinkButton
+                priority="default"
+                icon={<IconGithub />}
+                href="https://github.com"
+              >
+                {t('Sign in with GitHub')}
+              </LinkButton>
+            </Prereq>
+            <Prereq>
+              <PrereqMainText>{t('Install the GitHub Sentry App')}</PrereqMainText>
+              <PrereqSubText>
+                <Link to="">{t('Install the app')}</Link>
+                {t(
+                  " on your GitHub org in your Sentry org. You will need to be an Owner of your GitHub organization to fully configure the integration. Note: Once linked, a GitHub org/account can't be connected to another Sentry org."
+                )}
+              </PrereqSubText>
+            </Prereq>
+            <Prereq>
+              <PrereqMainText>
+                {t('Connect your GitHub identities in Sentry')}
+              </PrereqMainText>
+              <PrereqSubText>
+                {t('In your Sentry ')}
+                <Link to="https://sentry.io/settings/account/identities">
+                  {t('identities')}
+                </Link>
+                {t(
+                  " settings, link your GitHub account to your profile. If you're having trouble adding the integration, "
+                )}
+                <Link to="https://sentry.io/settings/account/identities">
+                  {t('disconnect')}
+                </Link>
+                {t(' disconnect then ')}
```

Comment:
`m`: There's a double disconnect here, I think we need to remove the second one 👀